extensions/amp-bind/0.1/bind-expr-defines.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * A single node in the AST of a `BindExpression`.
+ * @typedef {{
+ *   type: ASTNodeType,
+ *   args: Array<ASTNode>,
+ *   value: (ASTNodeValue|undefined),
+ * }}
+ */
+export let ASTNode;
+
+/**
+ * Type of a node in the AST of a `BindExpression`.
+ * @enum {string}
+ */
+export const ASTNodeType = {
```

Comment:
Using a number here would reduce file size and save some memory usage. Is there a best practice regarding enum types?